handsontable/src/helpers/dom/element.js

```diff
@@ -1148,3 +1173,45 @@ export function runWithSelectedContendEditableElement(element, callback, invisib
 
   removeContentEditableFromElementAndDeselect(element, invisibleSelection);
 }
+
+/**
+ * Creates a div element and appends it to the parent element with the provided class name(s) and attributes.
+ *
+ * @private
+ * @param {HTMLElement | null} parentElement The parent element.
+ * @param {string} elementType The type of the new element.
+ * @param {Array[] | undefined} attributes An array containing the attributes to be added. Each element of the array
+ * should be an array in a form of `[attributeName, attributeValue]`.
+ * @param {string|Array} className Class name as string or array of strings.
+ * @returns {HTMLElement} The created div element.
+ */
+export function appendElement(parentElement, elementType, attributes = [], className = []) {
+  const element = parentElement.ownerDocument.createElement(elementType);
+
+  if (className) {
+    addClass(element, className);
+  }
+
+  if (attributes && attributes.length) {
+    setAttribute(element, attributes);
+  }
+
+  parentElement.appendChild(element);
+
+  return element;
+}
+
+/**
+ * Removes a child HTML element from the parent element. If the child element does not exist, nothing happens.
+ *
+ * @private
+ * @param {HTMLElement | null} parentElement The parent element.
+ * @param {HTMLElement | null} childElement The parent element.
+ */
+export function removeChildIfExists(parentElement, childElement) {
+  if (!parentElement || !childElement) {
+    return;
+  }
+
+  parentElement.removeChild(childElement);
+}
```

Comment:
I wonder if this helper is necessary ... I think that `parent.removeChild()` can be replaced by `child?.remove()` in the target location.